### DISCUSSION TITLE
How do iI view GhidraClass html docs?

### DISCUSSION THREAD

**Discussion by karpiyon** [NationalSecurityAgency/ghidra#7191] [Q&A] [answered]
Hi,
How do I view the docs under:
https://github.com/NationalSecurityAgency/ghidra/blob/master/GhidraDocs/GhidraClass/Intermediate/HeadlessAnalyzer.html

In github it does not look like html page?
Should I download everything?
Can I open it from within Ghidra?

Thanks

**Comment by ryanmkurtz** [comment, marked as answer]
You can either clone the repository and open the local html files with your web browser, or you can find them in a Ghidra release under the top-level `docs/` directory.

**Comment by karpiyon** [comment]
Thanks
